sse2neon.h

```diff
@@ -5163,6 +5163,29 @@ FORCE_INLINE __m128d _mm_cmpnge_sd(__m128d a, __m128d b)
 // https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_cmpngt_pd
 #define _mm_cmpngt_pd(a, b) _mm_cmple_pd(a, b)
 
+// Compare packed double-precision (64-bit) floating-point elements in a and b
+// to see if neither is NaN, and store the results in dst.
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_cmpord_pd
+FORCE_INLINE __m128d _mm_cmpord_pd(__m128d a, __m128d b)
+{
+#if defined(__aarch64__)
+    // Note: NEON does not have ordered compare builtin
```

Comment:
The comment was a bit misleading. Improve it to reflect the code.

Reply:
How about `For quiet NaN, w NaNs are not equal to each other, which is NaN_1 != NaN_2`

Reply:
It is fine to me. Append more grammatically correct descriptions.

Reply:
I was not aware of the change against the above comments.

Reply:
my fault. I have amended it.